### ISSUE TITLE
Add CRA Example

### ISSUE THREAD

**Issue by codingwithmanny** [Developer-DAO/web3-ui#245]
**Context:**

After trying to setup `web3-ui` locally with [CRA](https://create-react-app.dev).

**Proposal:**

Organize the examples folder to separate the different react platforms you can use to serve react with.

**_NOTE:_** This is just CRA as an example and then adding more and more in different issues/pull requests.

_Example:_ Proposed folder structure.

```diff
- example
+ examples
+  - nextjs
+  - vitejs
+  - cra
+  - gatsby
```

**Goal:**

Give more examples of how to use `web3-ui` with different libraries and frameworks.

**Additional Details:**

Related to #242

**Comment by codingwithmanny** [comment]
@e-roy Has mentioned he would like to take this on.

**Comment by e-roy** [comment]
I'll replicate the ui from the [PR](https://github.com/Developer-DAO/web3-ui/pull/244) when it gets merged if that works

**Comment by codingwithmanny** [comment]
@e-roy you still up for the CRA implementation?

**Comment by e-roy** [comment]
@codingwithmanny so I created a  [fork](https://github.com/e-roy/web3-ui/tree/cra-example)

I have a few questions on this before creating a PR

1. Do the examples need to be in the workspaces?  
I had trouble getting packages to load properly using workspaces.  I noticed both of the other examples had yarn.lock files.

2. I replicated the vitejs example for this.  The only difference is in the vite example, useWriteContract was used from the core package and I used useContract from the hooks package.  It looks like the change to useWrieContract from [refactor](https://github.com/Developer-DAO/web3-ui/commit/1bb2aa67f533a8699c4dcd14381a71fb7e495c7f#diff-37008fa4a176034754370e62d3a5c1a93cc0376e004b1a7434ecb7b9bcaea02a) isn't in the newest npm, so I used the older useContract.  

What are your thoughts on this?
